### PR TITLE
Changes to auto we are using, included start positions and cone scoring possibility

### PR DESCRIPTION
**Pull request by jherrmann06**

### CODE REVIEW

File: src/main/java/frc/robot/util/AutoGenerator.java [hunk 1]
```diff
@@ -58,6 +59,14 @@ private StartLocation(int value) {
     public AutoGenerator(DrivetrainSubsystem drivetrainSubsystem, FloorArmSubsystem floorArmSubsystem, ArmSubsystem armSubsystem, Intake intake, FloorIntake floorIntake) {   
         SwerveDriveKinematics kinematics = drivetrainSubsystem.getKinematics();
 
+        eventMap.put("scoreLow", new ScoreLow(drivetrainSubsystem, floorArmSubsystem, intake));
+        eventMap.put("scoreConeHigh", new ExtendAndScoreCone(drivetrainSubsystem, floorArmSubsystem, armSubsystem, intake));
+        eventMap.put("scoreCubeHigh", new RotateAndScoreCube(drivetrainSubsystem, floorArmSubsystem, armSubsystem, floorIntake));
+        eventMap.put("autoBalance", new PrintCommand("Auto balance begin"));
+        eventMap.put("beginCollectCube", new BeginCollectCube(floorArmSubsystem, floorIntake));
+        eventMap.put("beginCollectCube", new PrintCommand("========================beginCollectCube==================="));
```
Comment (jdlanning) [inline src/main/java/frc/robot/util/AutoGenerator.java]: I think this print command is disabling the begin collect cube event

File: src/main/java/frc/robot/commands/groups/ScoreLow.java [hunk 2]
```diff
@@ -4,19 +4,21 @@
...
 
-  public ScoreCubeLow(DrivetrainSubsystem drive, FloorArmSubsystem floorArm, Intake intake) {
+  public ScoreLow(DrivetrainSubsystem drive, FloorArmSubsystem floorArm, Intake intake) {
     addCommands(
       new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_SHOOTING_POSITION),
       new WaitCommand(0.5),
       new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT_POSITION),
-      new DropCube(intake).withTimeout(0.5)
+      new DropCube(intake).withTimeout(0.5),
+      new DropCone(intake).withTimeout(0.5)
```
Comment (jdlanning) [inline src/main/java/frc/robot/commands/groups/ScoreLow.java:21]: I have some concern for the cone case of I taking too far if trying to drop the cube. We might eventually need two commands

File: src/main/java/frc/robot/commands/groups/EndCollectCube.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import frc.robot.commands.floorArm.FloorMoveArmToPostion;
+import frc.robot.commands.intake.floor.FloorPickup;
+import frc.robot.subsystems.FloorArmSubsystem;
+import frc.robot.subsystems.FloorIntake;
+
+public class EndCollectCube extends SequentialCommandGroup {
+
+  public EndCollectCube(FloorArmSubsystem floorArm, FloorIntake intake) {
+    addCommands(
+      new FloorPickup(intake).withTimeout(0),
```
Comment (jdlanning) [inline src/main/java/frc/robot/commands/groups/EndCollectCube.java]: This line doesn’t seem needed

**Review by jdlanning — COMMENTED** [review]
Found a couple of things for you to look at

**Comment by jherrmann06** [comment]
Responded to those comments

**Review by jdlanning — APPROVED** [review]

**Pull request merged**